lib/iphone-simulator-xcode-simctl.ts

```diff
@@ -229,15 +234,18 @@ export class XCodeSimctlSimulator extends IPhoneSimulatorNameGetter implements I
 		}
 
 		if (!this.isDeviceBooted(device)) {
-			let bootedDevice = this.getBootedDevice();
-			if (bootedDevice && bootedDevice.id !== device.id) {
-				this.killSimulator();
+			let isSimulatorAppRunning = this.isSimulatorAppRunning(),
```

Comment:
both variables should be `const`s instead of `let`. 
Also we prefer declaring each of them separately, i.e.:
```TypeScript
const isSimulatorAppRunning = this.isSimulatorAppRunning();
const haveBootedDevices = this.haveBootedDevices();
```